pytest_fixtures/broker.py

```diff
@@ -9,6 +9,13 @@
 from robottelo.hosts import Satellite
 
 
+@pytest.fixture(scope='session')
+def default_sat():
```

Comment:
With this being introduced here, I'm less concerned about priority for destructive tests.